qemu/tests/qemu_disk_img.py

```diff
@@ -176,6 +177,32 @@ def verify_info(self, params=None):
                 msg += "Expect: %s, actual: %s" % (evalue, avalue)
                 raise error.TestFail(msg)
 
+    @error_context.context_aware
+    def check_backingfile(self):
+        error_context.context("check image('%s') backing file" %
+                              self.image_filename, logging.info)
+        out = self.get_info()
+        if not out:
+            msg = "Fail to get image('%s') info" % self.image_filename
+            raise error.TestFail(msg)
+        backingfile = re.search(r'backing file: +(.*)', out, re.M)
```

Comment:
try:
      backingfile =  re.search(r'backing file: +(.*)', out, re.M).group(1)
      if backingfile != self.base_image_filename:
            raise error.TestFailed(xxxx)
 except TypeError:
      if  not (self.base_tag != "null" and  self.base_tag):
            raise